src/TimeoutMigrationTool/Program.cs

```
@@ -184,6 +188,21 @@ static int Main(string[] args)
             return app.Execute(args);
         }
 
+        static DateTime GetCutoffTime(CommandOption cutoffTimeOption)
+        {
+            DateTime cutoffTime;
+            if (!cutoffTimeOption.HasValue())
+            {
+                cutoffTime = DateTime.UtcNow;
+            }
+            else if (!DateTime.TryParse(cutoffTimeOption.Value(), out cutoffTime))
+            {
+                Console.WriteLine($"Unable to parse the cutofftime, please supply the cutoffTime in the following format '{CutoffTimeFormat}'");
```

Should this throw instead? IF not we'll return an undefined date right?